### ISSUE TITLE
`onChange` is not fired if reset `html` and then input the a char same as before resetting.

### ISSUE THREAD

**Issue by ladjzero** [lovasoa/react-contenteditable#69]
The use case is like some IM app, type, send, clear the input, and type a next message.

If send an 'a', clear the input immediately. Later type an 'a', the onChange is not fired. 

Code and log below.

```
import React, { Component } from 'react';
import ContentEditable from 'react-contenteditable';

class App extends Component {
  state = {
    html: '',
  }

  render() {
    const { html } = this.state;

    return (
      <div className="App">
        <ContentEditable
          html={html}
          onChange={e => {
            console.log('onChange', e.target.value);
            this.setState({ html: e.target.value });
          }}
        />
        <button onClick={() => {
          console.log('send', html);
          this.setState({ html: '' });
        }}>send</button>
      </div>
    );
  }
}

export default App;

```

![selection_699](https://user-images.githubusercontent.com/4110311/30947378-076f92e6-a43c-11e7-85c8-218be3510641.png)

**Comment by ladjzero** [comment]
```
 componentDidUpdate() {
    if ( this.htmlEl && this.props.html !== this.htmlEl.innerHTML ) {
      // Perhaps React (whose VDOM gets outdated because we often prevent
      // rerendering) did not update the DOM. So we update it manually now.
      this.htmlEl.innerHTML = this.props.html;
    }
  }
```

Or `this.lastHtml = this.htmlEl.innerHTML = this.props.html;` could fix it. I'm not sure.

**Comment by lovasoa** [comment]
this should have been fixed in 2fadc9089299a79797642dbacdc5dcced561946d

Can you check?
